valhalla/jawn/src/lib/handlers/LytixHandler.ts

```diff
@@ -33,7 +35,11 @@ export class LytixHandler extends AbstractLogHandler {
       modelOutput: JSON.stringify(context.processedLog.response.body) ?? "",
       metaData: context.processedLog.request.properties ?? {},
       session:
-        context.message.log.request.properties?.["Helicone-Session-ID"] ?? "",
+        context.message.log.request.properties?.["Helicone-Session-Id"] ?? "",
```

Comment:
We are still seeing `""` here, maybe it's because of this `ID` vs `Id`? 

Saw its `Id` in other places in the codebase.